etcdserver/v3_server.go

```diff
@@ -279,7 +279,7 @@ func (s *EtcdServer) LeaseRenew(ctx context.Context, id lease.LeaseID) (int64, e
 	defer cancel()
 
 	// renewals don't go through raft; forward to leader manually
-	for cctx.Err() == nil && err != nil {
```

Comment:
Good catch. Can we add a log saying `renewing lease...`?

Reply:
Where do you want the log added ?
We have nested loop - I don't want to spam the log file.

Reply:
@gyuho 
Please advise. I wonder if this can go in without adding log.

Reply:
ping @gyuho